k8s/upgrades/0.8.0-0.8.1/README.md

```diff
@@ -0,0 +1,106 @@
+# UPGRADE FROM OPENEBS 0.8.0 TO 0.8.1
+
+## Overview
+
+This document describes the steps for upgrading OpenEBS from 0.8.0 to 0.8.1 
+
+The upgrade of OpenEBS is a two step process: 
+- *Step 1* - Upgrade the OpenEBS Operator 
+- *Step 2* - Upgrade the OpenEBS Volumes from previous versions (0.8.0) 
+
+### Terminology
+- *OpenEBS Operator : Refers to maya-apiserver & openebs-provisioner along w/ respective services, service a/c, roles, rolebindings*
+- *OpenEBS Volume: Storage Engine pods like cStor or Jiva controller(aka target) & replica pods*
+
+## Prerequisites
+
+*All steps described in this document need to be performed on the Kubernetes master or from a machine that has access to Kubernetes master*
+
+### Download the upgrade scripts
+
+The easiest way to get all the upgrade scripts is via git clone.
+
+```
+mkdir upgrade-openebs
+cd upgrade-openebs
+git clone https://github.com/openebs/openebs.git
+cd openebs/k8s/upgrade/0.8.0-0.8.1/
+```
+
+## Step 1: Upgrade the OpenEBS Operator
+
+### Upgrading OpenEBS Operator CRDs and Deployments
+
+The upgrade steps vary depending on the way OpenEBS was installed, select one of the following:
+
+#### Install/Upgrade using kubectl (using openebs-operator.yaml )
+
+**The sample steps below will work if you have installed openebs without modifying the default values in openebs-operator.yaml. If you have customized it for your cluster, you will have to download the 0.8.1 openebs-operator.yaml and customize it again**
+
+```
+# Starting with OpenEBS 0.6, all the components are installed in namespace `openebs`
+# as opposed to `default` namespace in earlier releases. 
+# If Upgrading from 0.5.x, delete older operator. 
+#kubectl delete -f https://raw.githubusercontent.com/openebs/openebs/v0.5/k8s/openebs-operator.yaml
+# Wait for objects to be delete, you can check using `kubectl get deploy`
+
+#Upgrade to 0.8.1 OpenEBS Operator
+kubectl apply -f https://openebs.github.io/charts/openebs-operator-0.8.1.yaml
+```
+
+#### Install/Upgrade using helm chart (using stable/openebs, openebs-charts repo, etc.,) 
+
+**The sample steps below will work if you have installed openebs with default values provided by stable/openebs helm chart.**
+
+- Run `helm repo update` to update local cache with latest package
+- Run `helm ls` to get the release name of openebs. 
+- Upgrade using `helm upgrade -f https://openebs.github.io/charts/helm-values-0.8.1.yaml <release-name> stable/openebs`
+
+#### Using customized operator YAML or helm chart.
+As a first step, you must update your custom helm chart or YAML with 0.8.1 release tags and changes made in the values/templates. 
+
+You can use the following as references to know about the changes in 0.8.1: 
+- openebs-charts [PR#2314](https://github.com/openebs/openebs/pull/2314) as reference.
```

Comment:
this PR number is still valid for 0.8.1 also? I see the same number in 0.7 to 0.8 upgrade script also

Reply:
need to change it

Reply:
done